src/AddressRegistry.Api.Oslo/appsettings.json

```diff
@@ -0,0 +1,56 @@
+{
+  "ConnectionStrings": {
+    "Events": "Server=(localdb)\\mssqllocaldb;Database=EFProviders.InMemory.AddressRegistry;Trusted_Connection=True;",
+    "LegacyProjections": "Server=(localdb)\\mssqllocaldb;Database=EFProviders.InMemory.AddressRegistry;Trusted_Connection=True;",
+    "SyndicationProjections": "Server=(localdb)\\mssqllocaldb;Database=EFProviders.InMemory.AddressRegistry;Trusted_Connection=True;",
+    "BuildingLegacyProjections": "Server=(localdb)\\mssqllocaldb;Database=EFProviders.InMemory.AddressRegistry;Trusted_Connection=True;"
+  },
+
+  "DataDog": {
+    "Enabled": false,
+    "Debug": false,
+    "ServiceName": "AddressRegistry Oslo"
+  },
+
+  "BaseUrl": "https://api.staging-basisregisters.vlaanderen/",
+
+  "Configuration": {
+    "ConnectionString": "Server=(localdb)\\mssqllocaldb;Database=EFProviders.InMemory.AddressRegistry;Trusted_Connection=True;"
+  },
```

Comment:
Can't remember what this is used for? Any idea?

Reply:
nope it's a copy paste from legacy